src/arc.ts

```diff
@@ -19,37 +19,6 @@ let selectedProvider: any;
 let web3Modal: Web3Modal;
 let initializedAccount: Address;
 
-export type Networks = "main"|"rinkeby"|"ganache"|"xdai"
```

Comment:
why this is removed ?

Reply:
see the corresponding change and the comment where I explained in the code, line 136 in lib/util.ts

Reply:
is this change is a must to support kovan or any other network ?

Reply:
same answer as above because this is part of the same change.

Without that change we would be worsening the problem by adding a dependency to the circle.  There is less risk in fixing this than in not (which is easy to say because this change is virtually risk-free).

It is not a must for this PR, unless you're worried about the potential for webpack to behave badly because of circular module dependencies.